builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakAuthenticationFilter.java

```diff
@@ -65,13 +66,14 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
         HttpServletRequest request = (HttpServletRequest) req;
         String authHeader = request.getHeader("Authorization");
         String requestURI = request.getRequestURI();
+        String upgrade = request.getHeader("upgrade");
```

Comment:
will it be possible to hack / bypass keycloak auth by setting `Upgrade: WebSocket` header to the malicious request ?